src/index.js

```diff
@@ -11,12 +11,12 @@ export default function (context, {react, targets = {}, modules} = {}) {
           ...(false === modules) && {modules: false}
         }
       ),
-      react && require('babel-preset-react')
+      react && require('@babel/preset-react')
     ].filter(Boolean),
     plugins: [
       [restSpread, {useBuiltIns: true}],
       ...react
-        ? [require('babel-plugin-transform-class-properties'), require('babel-plugin-inline-react-svg').default]
```

Comment:
seems like once the svg plugin is updated, we might as well merge this

Reply:
That's fine with me.